packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.html

```diff
@@ -11,84 +11,130 @@
-                    onselectvalue={onSelectValue} 
-                    key={v.value} 
-                    item={v}
-                    active-parent={activeParent}>
-                  </c-quantic-category-facet-value>
+                  <li class={categoryFacetLiClass} key={v.value}>
+                    <c-quantic-category-facet-value onselectvalue={onSelectValue} item={v}>
+                    </c-quantic-category-facet-value>
+                  </li>
                 </template>
               </ul>
-            </li>
+            </fieldset>
           </template>
+
           <template if:true={isFacetSearchActive}>
             <template if:true={hasSearchResults}>
-              <template for:each={facetSearchResults} for:item="result">
-                <c-quantic-category-facet-value 
-                  onselectvalue={onSelectValue} 
-                  key={result.index}  
-                  item={result} 
-                  is-search-result="true">
-                </c-quantic-category-facet-value>
-              </template>
+              <fieldset>
+                <legend class="slds-assistive-text">{field}</legend>
+                <ul class="slds-size_1-of-1">
+                  <template for:each={facetSearchResults} for:item="v">
+                    <li class={categoryFacetLiClass} key={v.value}>
+                      <c-quantic-category-facet-value onselectvalue={onSelectValue} item={v} is-search-result="true">
+                      </c-quantic-category-facet-value>
+                    </li>
+                  </template>
+                </ul>
+              </fieldset>
             </template>
             <div class="slds-p-horizontal_x-small">
               <template if:false={hasSearchResults}>
                 <li class="slds-grid slds-var-m-top_x-small">
-                  <span class="slds-truncate slds-var-p-around_small slds-size_1-of-1 facet__search-results_no-match">{noMatchesForLabel}</span> 
+                  <span class="slds-truncate slds-var-p-around_small slds-size_1-of-1 facet__search-results_no-match"
+                    >{noMatchesForLabel}</span
+                  >
                 </li>
               </template>
               <template if:true={canShowMoreSearchResults}>
                 <li>
-                  <span class="slds-truncate facet__search-results_more-matches">{moreMatchesForLabel}</span> 
+                  <span class="slds-truncate facet__search-results_more-matches">{moreMatchesForLabel}</span>
                 </li>
               </template>
             </div>
           </template>
-        </ul>
+        </div>
         <div class="facet__show slds-m-top_x-small slds-p-horizontal_x-small">
           <template if:true={canShowLess}>
-            <button variant="base" label={labels.showLess}
-              class="facet__show-less slds-p-around_none" onclick={showLess}>
+            <button
+              variant="base"
+              aria-label="Show less"
```

Comment:
You don't need that `aria-label` since you have a span with a label inside the button's HTML